### PR TITLE
Report tests in TAP format

### PR DESCRIPTION
**Pull request by Nancy-Chauhan**
Create a new report.tap with test results written in TAP format
@stffrdhrn @oleg-nenashev

### CODE REVIEW

**Comment by stffrdhrn** [comment]
Why is it a draft? Looks ok to me

**Comment by Nancy-Chauhan** [comment]
@stffrdhr Please confirm if this looks okay to you for output in .tap format?
```
PASS >> ok
UNEXPECTED PASS >> not ok
TIME OUT >> not ok
UNEXPECTED FAIL >> not ok
FAIL >> not ok 
```

File: native/runtests.sh [hunk 1]
```diff
@@ -100,21 +120,26 @@ for test_path in $DIR/build/or1k/${TEST_PATTERN}; do
...
+        fail $test_name "UNEXPECTED PASS"
         ((unexpected_pass_count++))
       else
         echo -e "$PASS"
+        pass $test_name
       fi
     else
       if [ "$EXPECTED_FAILURES" ] && [[ "$EXPECTED_FAILURES" =~ $expected_failure_pattern ]] ; then
         echo -e "$FAIL"
+        fail $test_name
```
Comment (stffrdhrn) [inline native/runtests.sh]: This is the expected fail case. It should be ok.

**Comment by stffrdhrn** [comment]
I don't quite understand your comment.  But fail is actually ok.

**Comment by Nancy-Chauhan** [comment]
@stffrdhrn Does it look fine now? I will do changes in Jenkinsfile then.

**Pull request merged**